pkg/postage/batchservice/batchservice.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package batchservice
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/postage"
+)
+
+// BatchService implements the EventUpdater interface.
+type BatchService struct {
```

Comment:
type does not need to be exported, since the interface is being returned from the constructor

Reply:
Unexported type.